### ISSUE TITLE
Issue that some components are not available in Next.js v15

### ISSUE THREAD

**Issue by NekoSakuraLucia** [rxy001/nex-ui#1]
The problem this time is that some components cannot be used in Next.js. Module export error.

**Comment by rxy001** [comment]
Did you install it from npm? There is currently no available npm package released yet.

**Comment by NekoSakuraLucia** [comment]
> Did you install it from npm? There is currently no available npm package released yet.

No, install it via the npm link in the folder I cloned yours from and just test it.

**Comment by rxy001** [comment]
```
<NexProvider>
    <Button>123</Button>
</NexProvider>
```

**Comment by rxy001** [comment]
I've tested it locally and didn't find any errors. It might be a configuration issue with Next.js.

If you want to test the built package：

1. `pnpm run build`
2. Import the `NexProvider` at the entry point

If you want to test the source code, you can configure it through webpack's alias or conditionNames.

If there are any issues, please provide more information. Thank you.

**Comment by NekoSakuraLucia** [comment]
> I've tested it locally and didn't find any errors. It might be a configuration issue with Next.js.
> 
> If you want to test the built package：
> 
> 1. `pnpm run build`
> 2. Import the `NexProvider` at the entry point
> 
> If you want to test the source code, you can configure it through webpack's alias or conditionNames.
> 
> If there are any issues, please provide more information. Thank you.

It's working now. thank
